Tests/test_excinfo.py

```diff
@@ -7,7 +7,7 @@
 import sys
 import unittest
 
-from iptest import is_cli, run_test
+#from iptest import is_cli, run_test
```

Comment:
Might as well just delete the code instead of commenting it out.

Reply:
I'm ok with using unittest directly. Makes it easier to run against CPython. Not really sure what we gain by using `run_test`? If it's a problem then it's easy to change it back.